### PR TITLE
Add virtual field support to slug labels

### PR DESCRIPTION
**Pull request by jadb**

### CODE REVIEW

**Comment by dereuromark** [comment]
Maybe a test and this is good to go. Thanks for finding and fixing this.

**Comment by jadb** [comment]
Fails are unrelated to my changes :)

**Comment by dereuromark** [comment]
True. Some bc breaks in rc2 development.

But please use psr2r as cs.

**Comment by jadb** [comment]
don't have it installed - can you tell what needs to be changed? My IDE uses `.editorconfig` and noticed tabs were left as tabs (not spaces) - what needs to be fixed?

**Comment by dereuromark** [comment]
The consistent brace style. Just as cakephp always did.

**Comment by jadb** [comment]
Done, even though I really believe all plugins should stick with cake's adoption of the PSR and use the same CS (even though I don't whole-heartedly agree to everything in that PSR), for the sake of the community and of pushing wider standards' adoption (but that's just me :wink:).

File: tests/TestApp/Model/Entity/SluggedArticle.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+use Cake\ORM\Entity;
+
+class SluggedArticle extends Entity
+{
+	protected function _getSpecial()
+	{
+		return 'dereuromark';
+	}
+}
```
Comment (dereuromark) [inline tests/TestApp/Model/Entity/SluggedArticle.php]: This fils is also not psr2-r

**Comment by dereuromark** [comment]
Thank you :)

**Pull request merged**